### ISSUE TITLE
[Accessibility] disabled buttons cannot be focussed/tab

### ISSUE THREAD

**Issue by distantnative** [k-next/kirby#4932]
If a button in our Panel is disabled, e.g. the status flag icon/button for a status that cannot be changed, it is not focusable. Which is bad because then a screenreader can actually neither read/announce the current status nor that changing the status is disabled. For a user with screenreader, the status flag/button simple doesn't exist for those with disabled button.

**Comment by distantnative** [comment]
Will be included in upcoming v4-alpha.5
